packages/flutter/test/painting/text_style_test.dart

```diff
@@ -5,6 +5,7 @@
 import 'dart:ui' as ui show TextStyle, ParagraphStyle, FontFeature, Shadow;
 
 import 'package:flutter/painting.dart';
+import 'package:flutter/src/foundation/constants.dart';
```

Comment:
FWIW, it should just be enough to import `package:flutter/foundation.dart`

@goderbauer don't we have a lint that should have caught this? Or does it not apply to tests?